go-app-personal.js

```
@@ -108,8 +116,13 @@ go.app = function() {
                 ],
 
                 next: function(choice) {
+                    self.contact.extra.language_choice = choice.value;
+
                     return self.im.user.set_lang(choice.value)
                     .then(function() {
+                        return self.im.contacts.save(self.contact);
+                    })
+                    .then(function() {
```

let's cut a separate PR for future refactorings where we fail fast if `.save(contact)` fails. Currently we're assuming that'll always be ok.